README.md

```diff
@@ -170,6 +180,15 @@ $ cmake -DUSER_PROJECT_PATH=examples/flutter-drm-backend ..
 $ cmake --build .
 ```
 
+### Build for EGLStream backend
```

Comment:
Put it under the description of DRM.

Reply:
I moved it under the description of DRM.